Source/SwiftLintFramework/Rules/FirebaseConfigFetchRule.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  FirebaseConfigFetchRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigFetchRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_fetch",
+        name: "Firebase Config Fetch",
+        description: "Firebase Config fetch should be called in viewDidLoad().",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+            "        (status, error) -> Void in \n" +
+            "    }\n" +
+            "  }\n" +
+            "}"
+        ],
```

Comment:
Please consider adding these two examples:

```swift
            "class ViewController: UIViewController {\n" +
                "  override func viewDidLoad() {\n" +
                "    super.viewDidLoad()\n" +
                "  }\n" +
            "}",
            "class ViewController: UIViewController { }",
```

The first one will make sure an "empty" `viewDidLoad()` won't trigger a violation. The second one will make sure a View Controller that doesn't override `viewDidLoad()` won't trigger a violation as well.